src/analyzers/Infeasibility.jl

```diff
@@ -5,11 +5,57 @@
 
 module Infeasibility
 
-import ModelAnalyzer
 import MathOptInterface as MOI
+import ModelAnalyzer
+
+# This type and the associated function were inspired by IntervalArithmetic.jl
+# Copyright (c) 2014-2021: David P. Sanders & Luis Benet
+struct Interval{T<:Real}
+    lo::T
+    hi::T
+
+    function Interval(lo::T, hi::T) where {T<:Real}
+        @assert lo <= hi
+        return new{T}(lo, hi)
+    end
+end
+
+Base.convert(::Type{Interval{T}}, x::T) where {T<:Real} = Interval(x, x)
+
+Base.zero(::Type{Interval{T}}) where {T<:Real} = Interval(zero(T), zero(T))
+
+Base.iszero(a::Interval) = iszero(a.hi) && iszero(a.lo)
+
+function Base.:+(a::Interval{T}, b::Interval{T}) where {T<:Real}
+    return Interval(a.lo + b.lo, a.hi + b.hi)
+end
+
+function Base.:*(x::T, a::Interval{T}) where {T<:Real}
+    if iszero(a) || iszero(x)
+        return Interval(zero(T), zero(T))
+    elseif x >= zero(T)
+        return Interval(a.lo * x, a.hi * x)
+    end
+    return Interval(a.hi * x, a.lo * x)
+end
```

Comment:
It's questionable that a very simple type like this and a few methods need a license attribution. Since both packages are MIT, we're in the clear, so I just moved into this file.